ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
re:

>I would like to have some place for ci-independent build scripts that will not get contaminated with our ci-specific stuff to the point that it would be impossible to use them outside of our ci.

Keeping `ci/bamboo/` as simple as possible is a high priority for me, and I should hope others, too.  For example, `build-osx.sh` reads like an example that a user would copy+paste.  If "contamination" is a concern, that ought to be a red flag that it's getting too complicated.  

Alas, https://github.com/vitaly-krugl/nupic.core/blob/b9e50ce38a1c4487631af04c148902946d708c10/scripts/manylinux/build_manylinux_wheel.sh is way more complicated than https://github.com/vitaly-krugl/nupic.core/blob/b9e50ce38a1c4487631af04c148902946d708c10/ci/bamboo/build-linux-release.sh but they effectively do the same thing.  The primary difference being different assumptions about _where_ they are being run.  The former, in our custom manylinux container, the latter in a stock ubuntu 14.04 container.